### ISSUE TITLE
VuePress Migration

### ISSUE THREAD

**Issue by bencodezen** [vuejs-fr/vuejs.org#188]
Hello! My name is Ben Hong and I'm a Vue.js Community Partner that is currently working with the Vue.js Core Docs Team to help with updating the platform from Hexo to VuePress. During this time, we plan to have a content freeze in order to ensure all of the sites stay in sync with one another.

As the maintainers for the French translation site, are there any pain points or concerns that you might have about this migration? We would love to hear from you and make sure your thoughts are taken into consideration before starting work on this.

Looking forward to your response!

**Comment by Haeresis** [comment]
Bonjour Ben,

Understood.

Can you indicate me the last commit I can use to update French website with
Hexo before the documentation was freeze?

Cordialement,

Bruno

---

Réponses rapides :

Bien reçu
<bruno.lesieur@gmail.com?subject=Message%20bien%20re%C3%A7u.&body=J%27ai%20bien%20pris%20note%20de%20votre%20dernier%20email.%20Merci%20%21>
OK
<bruno.lesieur@gmail.com?subject=C%27est%20OK%20pour%20moi.&body=Oui,%20%C3%A7a%20me%20va%20bien.%20Merci%20%21>
Non
<bruno.lesieur@gmail.com?subject=Non,%20merci.&body=Merci,%20mais%20%C3%A7a%20ne%20va%20pas%20%C3%AAtre%20possible.>
ASAP
<bruno.lesieur@gmail.com?subject=Je%20reviens%20vers%20vous%20au%20plus%20vite.&body=J%27ai%20bien%20pris%20note%20de%20votre%20dernier%20email,%20je%20reviens%20vers%20vous%20au%20plus%20vite.>

[image: Bruno Lesieur] <http://www.lesieur.name/>
[image: Logo Google+] <https://plus.google.com/+BrunoLesieur/> [image: Logo
Twitter] <https://twitter.com/MachinisteWeb> [image: Logo LinkedIn]
<https://www.linkedin.com/in/brunolesieur/>
Bruno Lesieur
Développeur web Front-end et Node.js
Email : bruno.lesieur@gmail.com
Site : https://www.lesieur.name/

**Comment by bencodezen** [comment]
Hey Bruno!

We have not frozen the content yet. We just want to give everyone a chance to give feedback if there are any concerns about the migration / process. 

Does that help to clarify my initial message?

**Comment by bencodezen** [comment]
Hello! Just wanted to let you know that v3 docs are ready to start translating.

https://github.com/vuejs/docs-next

If you have any questions, please feel free to file an issue so we can help however we can!

**Comment by Haeresis** [comment]
Amazing! Any instruction about how proceed for translation from now?

**Comment by bencodezen** [comment]
Oops. I need to update that. That is no longer relevant.

I would do a fork and keep it separate from the v2 codebase since it's on VuePress instead of Hexo. As far as keeping it up to date, that's a trickier part. I know certain teams are using GitHub actions to track major changes, I'll try and see if I can get more info on this.
